.golangci.yml

```diff
@@ -77,9 +77,30 @@ linters-settings:
     #  2. you use go >= 1.10
     #  3. you do repeated runs (false for CI) or cache $GOPATH/pkg or `go env GOCACHE` dir in CI.
     use-installed-packages: false
-  golint:
-    # minimal confidence for issues, default is 0.8
-    min-confidence: 0.8
+  revive:
+    confidence: 0.8
+    severity: warning
+    errorCode: 0
+    warningCode: 0
+    rules:
+      - name: blank-imports
```

Comment:
These are the golint defaults. They are defined one-by-one for clarity on what's being checked.
I removed `dot-import` rule. The rest I fixed. If we decide, we can enable some additional rules.